### PR TITLE
Removing NEWS from news section, adding a link to weather

### PR DESCRIPTION
**Pull request by ponchorage**
We removed the NEWS from above the news section and added a link to weather. Weather icon is not showing up for everybody for some reason (need to investigate) and so adding a link to make sure everyone can get to it. Also change Home link to say News.

### CODE REVIEW

**Comment by janiv** [comment]
Looks like the addition of the weather text messes up the menu on certain screen sizes. Can you fix?

screen shot 2015-07-08 at 3 11 23 pm

**Comment by ponchorage** [comment]
What browser are you using?

File: src/props/default.js [hunk 1]
```diff
@@ -25,10 +25,10 @@ var props = {
     hrefBase: "//www.ksl.com",
     items: {
       news: {
-        text: "News",
+        text: "\xA0",
```
Comment (janiv) [inline src/props/default.js]: Let's put a comment here explaining why this is needed.

**Comment by ponchorage** [comment]
I just tried Chrome, Safari, and Firefox and couldn't reproduce the weather overlap issue.

**Comment by janiv** [comment]
Browser shouldn't matter.

Chrome:
screen shot 2015-07-08 at 3 11 23 pm

Firefox:
screen shot 2015-07-08 at 3 14 47 pm

Safari:
screen shot 2015-07-08 at 3 15 00 pm

**Comment by janiv** [comment]
@travislucas The KSL news team wanted to clarify what "Home" meant in the hamburger menu of the new KSL header. Here's a screenshot of the old menu:

screen shot 2015-07-08 at 3 18 00 pm

To clarify it, they renamed "Home" to "News" and removed the News header. Here's how the new design looks:

screen shot 2015-07-08 at 3 17 03 pm

Thoughts or feedback on how to improve this? I feel like without the blue News header its balance is off.

**Comment by stephanbergen** [comment]
I agree that the balance is off, but this is more any issue of usability than design at this point. "Home" wasn't working a good description, especially given what was in the submenu. "News" as a header doesn't really accurately describe everything listed under it, as much of it isn't weather. I'm open to suggestions...

**Comment by ponchorage** [comment]
@janiv Do you see the overlap when you got to this URL? http://www.ksl.com/?beta2015=1
We already implemented these changes and are passing them in through props. However, they cause you to see the NEWS and no Weather link at first and then the change happens when React render kicks in.

**Comment by stephanbergen** [comment]
@janiv Your seeing overlap because of the additions of "Shops" to marketplace. Shops currently isn't live and I haven't heard a timeline of when it will be..

**Comment by janiv** [comment]
Yes, it does look like turning on shops causes the issue but we do need to address it. http://shops.ksl.com/ is already running and they're using our header with shops turned on. We need to make sure we don't break existing functionality and allow us to turn on shops without any issue.

**Comment by travislucas** [comment]
I agree that it looks odd without a section header now. It almost looks like a mistake. Do any of these options work?
How about this for a section header (was blue NEWS)
Headlines
News Sections
Sections
News Categories
Categories

Or leave News (in blue) and change Home to:
News Home (I think I like this option the best)
Front Page
Home Page

**Comment by stephanbergen** [comment]
@travislucas sections or categories might work. Changing News to news home further exacerbates  the overlap issue.

**Comment by ponchorage** [comment]
Looking at it. I don't *think* fixing the overlap should be too bad. It's hopefully just a quick matter of changing the breakpoints.

**Comment by janiv** [comment]
Yeah, should just be tweaking the breakpoints a bit.

**Comment by ponchorage** [comment]
@janiv How are you testing header changes in your local dev environment?

**Comment by janiv** [comment]
I have header-dev.ksl.com setup that points to the example/build folder as the site root.

**Comment by travislucas** [comment]
1 ksl-header-2-production-2

See what you guys think about this.

**Comment by ponchorage** [comment]
That doesn't represent the subnav. That's where part of the issues lies. We have a bunch of other subsections under that and it's hard to come up with anything other than "News" that accurately describes those sections.

**Comment by stephanbergen** [comment]
@travislucas Front page isn't a good descriptor of what is under it (subnav). They are news sections. And the other sections, Traffic, Live, TV and Radio,  Brandview aren't really news

**Comment by stephanbergen** [comment]
@travislucas I have suggested adding Menu or replacing the hamburger with "Menu". I think something along those lines would be wise.

**Comment by ponchorage** [comment]
Adding in the Weather and Shops links pushes the breakpoint to 1070 (instead of 1024) for the "mobile" menu with the hamburger. That's just too wide to be dropping down to that view. We need to figure out something else here.

**Comment by janiv** [comment]
We probably need to add the medium design back in. It was removed due to issues with dropdowns on various android devices I believe.

**Comment by janiv** [comment]
Could we figure out why the weather icon is not showing instead?

**Comment by stephanbergen** [comment]
@janiv the dropdowns didn't work properly on iOS devices - like an iPhone 6+. There was no way to scroll the dropdown.  If we can get the weather widget to work 100% of the time, I am open to removing the weather link.

**Comment by ponchorage** [comment]
I just moved traffic under news. We should be good for now. I'm investigating why the weather icon doesn't show up for some.

**Comment by janiv** [comment]
Looks good for now. We'll address the hamburger menu and the section header name in another pull request.

**Comment by janiv** [comment]
@ponchorage Do you want to tag, make the release notes, and notify everyone?

**Comment by ponchorage** [comment]
Sure.

**Pull request merged**